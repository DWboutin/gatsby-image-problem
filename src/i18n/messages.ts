import flat from 'flat'

import enMessages from './lang/en'

const defaultMessages = {
  en: enMessages,
}

const messages: { en: Record<string, string> } = {
  en: flat(defaultMessages.en, {
    delimiter: '_',
  }),
}

export default messages
