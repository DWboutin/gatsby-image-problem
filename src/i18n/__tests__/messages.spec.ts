import messages from '../messages'

describe('INTL messages', () => {
  it('should match snapshot', () => {
    expect(JSON.stringify(messages)).toMatchSnapshot()
  })
})
