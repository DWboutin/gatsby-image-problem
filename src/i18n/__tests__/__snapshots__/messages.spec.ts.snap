// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`INTL messages should match snapshot 1`] = `"{"en":{"global_menu_solutions_title":"Solutions","global_menu_solutions_links_brokerage":"Brokerage","global_menu_solutions_links_mortgage":"Mortgage","global_menu_solutions_links_recruiting":"Recruiting","global_menu_solutions_links_relocation":"Relocation","global_menu_resources_title":"Resources","global_menu_resources_links_blog":"Blog","global_menu_resources_links_productUpdates":"Product Updates","global_menu_resources_links_customers":"Customers","global_menu_resources_links_press":"Press","global_menu_company_title":"Company","global_menu_company_links_careers":"Careers","global_menu_company_links_contact":"Contact","global_menu_company_links_aboutUs":"About us","global_menu_cta":"Get In Touch","components_pagination_previous":"Prev","components_pagination_next":"Next","components_salary_aYear":"a year","solutionCards_title":"Drive revenue {br}across your business units.","solutionCards_solution_brokerage_title":"Brokerage","solutionCards_solution_brokerage_description":"Generate, nurture and convert real estate opportunities.","solutionCards_solution_brokerage_buttonLabel":"Learn more","solutionCards_solution_mortgage_title":"Mortgage","solutionCards_solution_mortgage_description":"Uncover financing opportunities on and off your website.","solutionCards_solution_mortgage_buttonLabel":"Learn more","solutionCards_solution_relocation_title":"Relocation","solutionCards_solution_relocation_description":"Refer clients outside of your service area to your trusted partners.","solutionCards_solution_relocation_buttonLabel":"Learn more","solutionCards_solution_recruiting_title":"Recruiting","solutionCards_solution_recruiting_description":"Attract & recruit great talent right on your website.","solutionCards_solution_recruiting_buttonLabel":"Learn more","postExcerpt_learnMore":"Learn more {arrow}","postExcerpt_classifications_blog":"Blog","postExcerpt_classifications_customers":"Customers","postExcerpt_classifications_careers":"Careers","postExcerpt_classifications_press":"Press","postExcerpt_classifications_product-updates":"Product Updates","customerStories_title":"Customer Stories","customerStories_readMore":"Read more stories","customerLogosSlider_title":"Trusted by","careers_latestJobsTitle":"Our last jobs","careers_otherJobsTitle":"Other roles","careers_experienceYear":"({years} {years, plural, =0 {years} one {year} other {years}})","careers_relatedPosts_title":"Related Jobs","careers_jobDetailsTitle":"Job Details","careers_salary":"Salary","careers_jobTypeTitle":"Job Type","careers_qualificationsTitle":"Qualifications","careers_jobType_full-time":"Full-time","careers_jobType_permanent":"Permanent","careers_jobType_contract":"Contract","careers_jobType_internship":"internship","careers_jobType_part-time":"Part-time","careers_jobType_remote":"Remote","blog_relatedPosts_title":"Related Articles","caseStudyHeading_previous":"← Previous case","caseStudyHeading_next":"Next case →","testimonialsSlider_title":"Testimonials","careerHeading_aYear":"a year","careerHeading_applyDirectly":"Apply Directly","contactPageForm_button":"Contact Sales","mailingListCtaSection_title":"Join our customer-obsessed readers list.","mailingListCtaSection_text":"Expertly curated content that’ll help you deliver an exceptional customer experience.","mailingListCtaSection_form_input_placeholder":"Your email address","mailingListCtaSection_form_button":"Subscribe","mailingListCtaSection_form_successMessage":"You've been added successfuly","demoFormCtaSection_title":"Get Your Demo now!","demoFormCtaSection_text":"Your privacy is important to us.","demoFormCtaSection_form_inputs_fullname_placeholder":"Full name","demoFormCtaSection_form_inputs_email_placeholder":"Email","demoFormCtaSection_form_inputs_phone_placeholder":"Phone","demoFormCtaSection_form_inputs_company_placeholder":"Company","demoFormCtaSection_form_inputs_jobTitle_placeholder":"Job title","demoFormCtaSection_form_button":"Get Your Demo","demoFormCtaSection_form_successMessage":"You've been added successfuly and someone will get back to you shortly","employeeListing_title":"Our dedicated team is targeting excellence and are results-driven","legalHeading_updated":"Updated: "}}"`;
